error-prone-contrib/src/test/resources/tech/picnic/errorprone/refastertemplates/ReactorTemplatesTestInput.java

```diff
@@ -95,6 +95,14 @@ Flux<Number> testFluxCast() {
     return Flux.just(1).map(Number.class::cast);
   }
 
+  Flux<Object> testFluxOnErrorComplete() {
+    return Flux.error(new IllegalStateException()).onErrorResume(e -> Flux.empty());
```

Comment:
We can simplify the example and as a result we don't need to use `Flux<Object>`, we can go with `Flux<Integer>`.